lib/valkyrie/persistence/active_fedora/query_service.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Valkyrie::Persistence::ActiveFedora
-  class QueryService
+  class QueryService < Valkyrie::Persistence::QueryService
```

Comment:
I want to continue trying to avoid inheritance, especially if it's just for a validation method

Reply:
How would you like it to be?

Reply:
Sorry for slowness! I hate to say it, but duplication?